File: scripts/pull-latest.py
import subprocess
import os

os.chdir('/vercel/share/v0-project')
result = subprocess.run(['git', 'pull', 'origin', 'main'], capture_output=True, text=True)
print("[v0] Git pull output:")
print(result.stdout)
if result.stderr:
    print("[v0] Git pull errors:")
    print(result.stderr)
print(f"[v0] Return code: {result.returncode}")

File: scripts/pull-latest.js
import { execSync } from 'child_process';

try {
  const result = execSync('git pull origin main', { cwd: '/vercel/share/v0-project', encoding: 'utf-8' });
  console.log('[v0] Git pull result:');
  console.log(result);
} catch (error) {
  console.error('[v0] Error pulling latest main:');
  console.error(error.message);
}
